src/main/resources/application.yml

```
@@ -129,6 +129,8 @@ rest:
     read-millis: "\${DEFAULT_REST_READ_TIMEOUT_MILLIS:60000}"
   organization:
     base-url: "\${ORGANIZATION_BASE_URL:http://p4pa-organization-microservice-chart:8080}"
+  workflow-hub:
+    base-url: "\${WORKFLOW_HUB_BASE_URL:http://p4pa-workflow-hub-microservice-chart:8080}"
```

Have you create the PR on aks-deploy repo to fill this variable?

ps: Please, do the same also for organization configuration
```suggestion
  organization:
    base-url: "\${ORGANIZATION_BASE_URL:http://localhost:8081}"
  workflow-hub:
    base-url: "\${WORKFLOW_HUB_BASE_URL:http://localhost:8082}"
```